packages/@aws-cdk/aws-redshift/test/integ.database.ts

```diff
@@ -32,6 +32,8 @@ const cluster = new redshift.Cluster(stack, 'Cluster', {
   encryptionKey: new kms.Key(stack, 'custom-kms-key'),
 });
 
+cluster.addToParameterGroup('enable_user_activity_logging', 'true');
```

Comment:
I'd like to see more test cases. One where the parameters are added directly in the constructor and one where addParameter() is used. Or rather, addParameters() if you take my suggestion above.

Reply:
I'll add some more tests to the `cluster.test.ts` for this. I don't *think* we need to update the integration test for this (unless we do change to `addParameters()`)